src/main/webapp/project/NewProject.js

```diff
@@ -214,7 +225,8 @@ class NewProject extends Component {
     let isDeterminationQuestionsValid = this.validateDeterminationQuestions();
     let isGeneralDataValid = this.validateGeneralData();
     let isInternationalCohortsValid = this.validateInternationalCohorts();
-    return isDeterminationQuestionsValid && isGeneralDataValid && isInternationalCohortsValid
+    let isDataSharingValid = this.validateDataSharing();
```

Comment:
I think this `let` and those above it can all be `const`s ... they're not getting re-assigned before the return.